### PR TITLE
DAOS-16271 mercury: Add patch to check ep for null in UCX key resolve.

### PR DESCRIPTION
**Pull request by jgmoore-or**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+mercury (2.4.0~rc5-2) unstable; urgency=medium
+  [ Joseph Moore ]
+  * Add patch to na_ucx.c to check ep in key_resolve
+
```
Comment (soumagne) [inline debian/changelog:4]: you're missing the date in the Debian log

Comment (jgmoore-or) [reply debian/changelog:4]: Fix it.

**Review by soumagne — APPROVED** [review]

**Review by frostedcmos — APPROVED** [review]

**Comment by jgmoore-or** [comment]
This PR has failed 3 unrelated test in the DAOS fanout (unrelated since the failures were running OFI and this PR only affects UCX). I am requesting forced landing.

**Review by frostedcmos — CHANGES_REQUESTED** [review]
As @mchaarawi pointed out this needs to be updated to Release: 2

**Comment by soumagne** [comment]
oh wow thanks @mchaarawi for seeing that...

**Review by soumagne — APPROVED** [review]

**Review by frostedcmos — APPROVED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Comment by mchaarawi** [comment]
> Requesting forced landing.

since this was rerun and failed again, can you please list the failing tests and corresponding tickets if known.

**Comment by jgmoore-or** [comment]
Fail tests were:
FTEST_ior.IorInterceptMultiClient.4
FTEST_nvme.NvmeIo.1

Both failures occurred when running under ofi+verbs for the md-ssd test suite. This ticket only affects UCX interface within Mercury.

**Comment by mchaarawi** [comment]
please make sure to push follow on PRs to update the build.config for daos master and 2.6

**Pull request merged**